src/main/java/org/shipkit/auto/version/NextVersionPicker.java

```diff
@@ -33,8 +33,17 @@ String pickNextVersion(Optional<Version> previousVersion, VersionConfig config,
         }
 
         if (!config.getRequestedVersion().isPresent()) {
-            String tag = runner.run("git", "describe", "--tags").trim();
+            String tag;
             String result;
+
+            try {
+                tag = runner.run("git", "describe", "--tags").trim();
+            } catch (Exception e) {
+                result = "0.0.1-SNAPSHOT";
```

Comment:
I think that exception should be somehow utilized. By logging it perhaps.

Reply:
> I think that exception should be somehow utilized. By logging it perhaps.

Thanks for suggestion @koral-- ! You're right. I think logging the exception is fine herel. I add `log.info()` and `log.debug` as @mockitoguy proposed.